crates/miden-lib/asm/miden/account.masm

```diff
@@ -264,6 +264,39 @@ export.get_item
     # => [VALUE]
 end
 
+#! Gets an item from the account storage at its initial state (beginning of transaction).
+#! Panics if the index is out of bounds.
+#!
+#! Inputs:  [index]
+#! Outputs: [INIT_VALUE]
+#!
+#! Where:
+#! - index is the index of the item to get.
+#! - INIT_VALUE is the initial value of the item at the beginning of the transaction.
+#!
+#! Panics if:
+#! - the index of the requested item is out of bounds.
```

Comment:
Nit: I would remove the panics statement at the beginning of the docs and only keep the standard `Panics if` section.